app/lib/core/models/transporter_info.dart

```diff
@@ -119,16 +119,16 @@ class TransporterInfo {
         'trainingExpiryDate': trainingExpiryDate,
       };
 
-  String toString() => '''Name of transporting company: $companyName
-  Address: $companyAddress
-  Driver(s) name(s): ${driverNames.join(",")}
-  Province and License Plate number of the conveyance transporting the animals: $vehicleProvince, $vehicleLicensePlate
-  (including trailer): $trailerProvince, $trailerProvince
-  Conveyance or container last cleaned and disinfected date and time: ${DateFormat("yyyy-MM-dd hh:mm").format(dateLastCleaned)}
-  Place: $addressLastCleanedAt
-  Driver(s) have been briefed on the contingency plan?: ${driversAreBriefed ? 'Yes' : 'No'}
-  Driver(s) have received humane transport training?: ${driversHaveTraining ? 'Yes' : 'No'}
-  Training type: $trainingType, Expiry date: ${DateFormat("yyyy-MM-dd hh:mm").format(trainingExpiryDate)}''';
+  String toString() => '''\nName of transporting company: $companyName
+  \nAddress: $companyAddress
```

Comment:
Is this working the way you think it is? The new line and space before this `\n` read like they could be included. 

- [ ] Could you take a screenshot, or better yet, share the PDF here so we can see?